source/delegate/test/Delegate.js

```diff
@@ -1069,6 +1148,41 @@ contract('Delegate Integration Tests', async accounts => {
       )
     })
 
+    it('Gets a quote to sell 1 WETH where sender != signer, passes', async () => {
```

Comment:
did you want to add a test before this that tests the failure of having no signature?

Reply:
Its on line 953